src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -761,6 +764,9 @@ internal int RunCore(TextWriter consoleOutput, ErrorLogger errorLogger, Cancella
                                     testSymWriterFactory: null,
                                     diagnostics: diagnosticBag,
                                     metadataOnly: emitOptions.EmitMetadataOnly,
+                                    includePrivateMembers: emitOptions.IncludePrivateMembers,
```

Comment:
The idea would be to reduce the effect of the binary incompatibility previously mentioned.

---
In reply to: [110484002](https://github.com/dotnet/roslyn/pull/18339#discussion_r110484002) [](ancestors = 110484002)